### DISCUSSION TITLE
Списки карт

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#86] [Ideas]
Сделать возможность просмотреть все списки, в который есть та - или иная карта. С режимами аналогично

**Comment by forwork10** [comment]
Слишком затратно, и не имеет смысла. Темболее при запуске комнаты с картой прямо из списка, список отображается в углу экрана вместе с картой.

**Comment by kkohno** [comment]
Какраз нужно посмотреть - затрат может и не быть... Обдумаем. Только зачем это?
